e2e/features/Udacity.feature

```diff
@@ -29,15 +29,20 @@ Scenario: 2. The card details and the filter fields should work correctly
    Then the "Results" label should not be visible
     And the course count should equal to the remembered course count
 
-Scenario: 3. The card details, dropdown filter and filter fields should work correctly
+Scenario Outline: 3. The card details, dropdown filter and filter fields should work correctly
   Given the Udacity course page is opened
 
    When the "Select Program Details" dropdown is clicked
-    And the "Intermediate" skill level is selected
+    And the <level> skill level is selected
    Then the selected filters field should contain the following filters:
-        |Intermediate|
-    And the intermediate course level logo should be visible for all cards
-    And the "Intermediate" course level label should be visible for all cards
+        |<filterField>|
+    And the <logoLevel> course level logo should be visible for all cards
+    And the <level> course level label should be visible for all cards
+Examples:
```

Comment:
Please make this examples table more simple, 1 colounm is enough.
You can write the " characters into the steps and you can also use toLowerCase/toUpperCase function if necessary